rllib/models/specs/specs_jax.py

```diff
@@ -11,20 +11,20 @@
 
 
 @DeveloperAPI
-class JAXSpecs(TensorSpecs):
-    @override(TensorSpecs)
+class JAXTensorSpec(TensorSpec):
```

Comment:
Why call this JAXTensorSpec, but the others only `XYZSpec` (w/o the "tensor", e.g. "TFSpec")?

Reply:
changed everything to `XXXTensorSpec` to be more precise, we may have `XXXDistributionSpecs` down the line too.